### DISCUSSION TITLE
Show battery temperatures on overview page

### DISCUSSION THREAD

**Discussion by mr-manuel** [Louisvdw/dbus-serialbattery#754] [Show and tell]
I was looking for something to display the different temperature sensors of my battery on the overview page in GuiMods but found nothing. Therefore I realized it myself:

![temperature_pages_guimods](https://github.com/Louisvdw/dbus-serialbattery/assets/12592765/8a0ef294-364b-4f93-9718-18de313499ad)

If anyone is interested: [venus-os_dbus-mqtt-temperature](https://github.com/mr-manuel/venus-os_dbus-mqtt-temperature)

**Comment by andrewcrosse** [comment]
that's a good idea, i thought having cell difference would be helpful also.
